frank/make_figs.py

```diff
@@ -18,26 +18,30 @@
 #
 """This module generates figures for a Frankenstein fit and its diagnostics.
 """
+
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.gridspec import GridSpec
+import logging
```

Comment:
I'm leaving the logging here because I don't see it as part of the main library, so I don't mind polluting it.

Reply:
In general it's not polluting, it's placing a useful functionality in the appropriate sections of the code.